src/main/java/mods/railcraft/common/util/misc/Predicates.java

```diff
@@ -24,7 +25,13 @@
  *
  * @author CovertJaguar <http://www.railcraft.info>
  */
+@SuppressWarnings({"unchecked", "rawtypes"})
 public class Predicates {
+
+    private static final Predicate NON_NULL = Objects::nonNull;
+    private static final Predicate ALWAYS_FALSE = t -> false;
+    private static final Predicate ALWAYS_TRUE = t -> true;
```

Comment:
You are breaking the generics, keep your PRs focused. Ignoring the warnings is stupid.